app/build.gradle

```diff
@@ -47,6 +50,10 @@ android {
         }
     }
     namespace 'org.alphatilesapps.alphatiles'
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```

Comment:
I think these are here because I use methods like List.of() and String.strip() which aren't available in older versions of Java. 

I can definitely change the code to not require Java 11 or later. However, if we think its a good idea to make newer java features available for developers, it looks like as long as whoever builds the app is using gradle 4.0 or later we can configure gradle so that the newer java features won't cause a problem even for devices with the minimum SDK. 

https://developer.android.com/studio/write/java8-support#library-desugaring

Reply:
I misread this as Java Version 11 being the ideal version (target meaning ideal), rather than the minimum supported Java version for the target. This seems fine! 👍